osu.Server.BeatmapSubmission/Models/Database/osu_beatmapset_version_file.cs

```diff
@@ -0,0 +1,14 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+// ReSharper disable InconsistentNaming
+
+namespace osu.Server.BeatmapSubmission.Models.Database
+{
+    public class osu_beatmapset_version_file
+    {
+        public byte[] sha2_hash { get; set; } = new byte[32];
```

Comment:
Yeah that makes sense. That "surrogate vs natural primary key" thing is something everyone has different opinions on so I just picked the smallest-column-count option and figured I'd just wait for feedback.

Reply:
This particular thing has spawned a bit of a complication since `osu_beatmapset_version_file` is not as self-sufficient after adding a surrogate PK as it was with the natural key. Read through https://github.com/ppy/osu-server-beatmap-submission/commit/8fa08fc2bd9ff6ee59e0367e54daed528d5227f3 and see if you can sorta follow the not-great naming, or whether you see any possibility of improvement.